### PR TITLE
[FIX] Logos safe area

### PR DESCRIPTION
**Pull request by brunosquadros**
Closes #ISSUE_NUMBER

### CODE REVIEW

File: public/images/logo/icon.svg [hunk 1]
```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <svg width="512px" height="512px" viewBox="0 0 512 512" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
     <!-- Generator: Sketch 51 (57462) - http://www.bohemiancoding.com/sketch -->
-    <title>Icon  Copy</title>
+    <title>Icon</title>
     <desc>Created with Sketch.</desc>
     <defs></defs>
     <g id="Icon--Copy" stroke="none" stroke-width="1" fill="none" fill-rule="evenodd">
```
Comment (vynmera) [inline public/images/logo/icon.svg]: should probably remove --Copy here too

File: public/images/logo/icon.svg [hunk 2]
```diff
@@ -1,16 +1,14 @@
...
-                <path d="M110.48867,171.559023 C99.2955147,171.559023 90.2216692,162.515313 90.2216692,151.359331 C90.2216692,140.20335 99.2955147,131.15964 110.48867,131.15964 C121.681826,131.15964 130.755671,140.20335 130.755671,151.359331 C130.755671,162.515313 121.681826,171.559023 110.48867,171.559023 Z M175.738527,171.559023 C164.545372,171.559023 155.471526,162.515313 155.471526,151.359331 C155.471526,140.20335 164.545372,131.15964 175.738527,131.15964 C186.931683,131.15964 196.005529,140.20335 196.005529,151.359331 C196.005529,162.515313 186.931683,171.559023 175.738527,171.559023 Z M240.988385,171.559023 C229.795229,171.559023 220.721383,162.515313 220.721383,151.359331 C220.721383,140.20335 229.795229,131.15964 240.988385,131.15964 C252.18154,131.15964 261.255386,140.20335 261.255386,151.359331 C261.255386,162.515313 252.18154,171.559023 240.988385,171.559023 Z" id="circles" fill="#DB2323" fill-rule="nonzero"></path>
-            </g>
+    <g id="Icon--Copy" stroke="none" stroke-width="1" fill="none" fill-rule="evenodd">
```
Comment (vynmera) [inline public/images/logo/icon.svg]: should remove `-Copy` here

File: public/images/logo/logo-dark.svg [hunk 3]
```diff
@@ -1,31 +1,30 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<svg width="4152px" height="1595px" viewBox="0 0 4152 1595" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
+<svg width="1000px" height="182px" viewBox="0 0 1000 182" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
     <!-- Generator: Sketch 51 (57462) - http://www.bohemiancoding.com/sketch -->
-    <title>Logo</title>
+    <title>Artboard Copy</title>
```
Comment (vynmera) [inline public/images/logo/logo-dark.svg]: should remove ` Copy` here

Comment (vynmera) [reply public/images/logo/logo-dark.svg]: (there's some more in this file, in the `g` tags)

File: public/images/logo/logo.svg [hunk 4]
```diff
@@ -1,29 +1,29 @@
...
     <!-- Generator: Sketch 51 (57462) - http://www.bohemiancoding.com/sketch -->
-    <title>Logo - PB</title>
+    <title>Artboard</title>
     <desc>Created with Sketch.</desc>
     <defs></defs>
-    <g id="Logo---PB" stroke="none" stroke-width="1" fill="none" fill-rule="evenodd">
-        <g id="Group-2" transform="translate(588.000000, 532.000000)" fill="#FFFFFF" fill-rule="nonzero">
-            <g id="logo-type" transform="translate(721.000000, 87.000000)">
+    <g id="Artboard" stroke="none" stroke-width="1" fill="none" fill-rule="evenodd">
+        <g id="Group-2-Copy" transform="translate(0.000000, 1.000000)" fill="#FFFFFF" fill-rule="nonzero">
```
Comment (vynmera) [inline public/images/logo/logo.svg]: and here's a `Copy` too

**Comment by sampaiodiego** [comment]
I've changed the logo sizes and alignments, here's how it looks with my changes:
image

image

**Pull request merged**